app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,10 @@
+class TagsController < ApplicationController
+
+  def show
+    @tag = Tag.find_by(name: params[:name])
+
+    render json: @tag
```

Comment:
Going to need to include 'notes.tags' to force the serializer to go one level deeper.